Microsoft.Toolkit.Uwp.SampleApp/SamplePages/samples.json

```diff
@@ -541,6 +541,15 @@
         "Icon": "/Assets/Helpers.png",
         "DocumentationUrl": "https://raw.githubusercontent.com/Microsoft/UWPCommunityToolkit/master/docs/helpers/BackgroundTaskHelper.md"
       },
+	  {
+        "About": "AppPinManager help to eaily add the app shortcut in StartMenu or TaskBar",
+        "CodeFile": "AppPinManagerHelperCode.bind",
+        "CodeUrl": "https://github.com/Microsoft/UWPCommunityToolkit/blob/master/Microsoft.Toolkit.Uwp/Helpers/AppPinManager/AppPinManager.cs",
+        "DocumentationUrl": "https://raw.githubusercontent.com/Microsoft/UWPCommunityToolkit/master/docs/helpers/AppPinManager.md",
+        "Icon": "/Assets/Helpers.png",
+        "Name": "AppPinManager",
+        "Type": "AppPinManagerHelperPage"
+      },
```

Comment:
Make sure to add a "BadgeUpdateVersionRequired" field to specify that this helper requires the Creators Update

Reply:
Done